lib/split/experiment.rb

```diff
@@ -8,6 +8,7 @@ class Experiment
     attr_accessor :alternatives
     attr_accessor :alternative_probabilities
     attr_accessor :metadata
+    attr_accessor :version
```

Comment:
I'm actually confused about the difference between `Split::Experiment` versions and `Split::Alternative` versions. Is the version that an alternative uses the same as the version of the associated experiment? Can they diverge?

Reply:
The concept of alternative as redefined in this upgrade is a little rickety. The way alternatives were created in the unforked version of split doesn't exactly lend itself to multiple versions. Alternatives are always bound to the experiment of the same version. I will try to flesh this out better.

Reply:
Mind giving some details on how this was fleshed out?